easybuild/easyconfigs/l/LLVM/LLVM-9.0.1-GCCcore-8.3.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '9.0.1'
+
+homepage = "https://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/']
+sources = ['llvm-%(version)s.src.tar.xz']
+checksums = ['00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+    ('Python', '3.7.4'),
+]
+
+dependencies = [
+    ('ncurses', '6.1'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+# required to install extra tools in bin/
+configopts += '-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON -DLLVM_ENABLE_RTTI=ON '
+configopts += '-DCMAKE_BUILD_TYPE=Release '
```

Recently we added a specific option to automatically handle `BUILD_TYPE`
```suggestion
build_type = 'Release'
```